staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go

```diff
@@ -18,8 +18,10 @@ package filters
 
 import (
 	"errors"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
nit: move into its own import block in-between STD lib and kube since it's third party

Reply:
OK, for this pr.

Involve @logicalhan as well. 

As an alternative, I'd prefer to add a wapper for `prometheus.ExponentialBuckets` to `k8s.io/component-base/metrics` package.  After that, we import `k8s.io/component-base/metrics`  is enough.
How do you think?  If it is ok for you, I'd like to present a PR.

Reply:
Yeah we actually do intend to go in this direction, if you'd like to wrap the buckets functionality that would be terrific.